source/linux/Foundational_Components/Kernel/Kernel_Drivers/Camera/CSI2RX.rst

```diff
@@ -435,6 +455,43 @@ Enabling camera sensors
     To re-build the kernel with above changes you can refer to the
     :ref:`Users Guide <users-guide-kernel-config>`.
 
+.. ifconfig:: CONFIG_part_variant in ('AM62X')
+
+    CSI2RX testing details
+    ======================
+
+    Following sensors have been tested with the latest SDK.
+
+        +------------------------------------+----------------------------+----------------------+------------+
```

Comment:
As you have 2 tables back to back, I would encourage you to use csv-table - https://docutils.sourceforge.io/docs/ref/rst/directives.html#csv-table-1

It allows use to provide table names. In addition each table gets a unique ID. So, you can directly have a link to it.
Eg - https://software-dl.ti.com/processor-sdk-linux/esd/AM62X/10_00_07_04/exports/docs/devices/AM62X/linux/Linux_Performance_Guide.html#id6 points directly to Dhrystone Benchmarks

Refer - https://github.com/TexasInstruments/processor-sdk-doc/blob/master/source/devices/AM62X/linux/Linux_Performance_Guide.rst?plain=1#L67 to check how @praneethbajjuri uses it in `Linux_Performance_Guide`